src/main/java/ru/itis/services/WishListService.java

```diff
@@ -21,13 +21,14 @@ public WishListService(WishListRepository wishListRepository, UserService userSe
     }
 
     public WishList addNewWishList(String title, String token) {
-        User currentUser = userService.findUserByToken(token).get();
-        WishList newWL = WishList.builder()
-                .title(title)
-                .author(currentUser)
-                .build();
-        currentUser.getWishLists().add(newWL);
-        return wishListRepository.save(newWL);
+        if (userService.findUserByToken(token).isPresent()) {
+            User currentUser = userService.findUserByToken(token).get();
+            WishList newWL = new WishList(title, currentUser);
+            currentUser.getWishLists().add(newWL);
+            wishListRepository.save(newWL);
+            return newWL;
+        }
+        return WishList.getDefault();
```

Comment:
I don't think the default is appropriate in this case.

Reply:
fixed